### PR TITLE
Parse extract

### PR DESCRIPTION
**Pull request by cbovino**
Co-Authored by @alexbednarek4 

parseExtract.ts:

**tldr** In this file we created one large function that uses two helper functions to accomplish our goal of returning a parsed query from a string.

We export the large function to be used inside extension.ts

**Raw**
parseExtract function accepts a string, which in our cases is the document of the on save event. The function creates and returns an array full of the queries from the user's saved document that are wrapped in Trinity(). 

parseExtract findQuery is a helper function that gets recursively called until all of the queries inside the user's saved file are found.

parseExtract extract is a helper function that extracts the query and the remaining parts of a string after a query and returns the results in an array to findQuery.

The parseExtract function is being called in extension.ts, because we wanted to test the functionality. At this point, it just outputs an array of the queries on each user save. 

Let us know if you have any questions, we will be refactoring and commenting in more depth in the future.

### CODE REVIEW

**Review by alexbednarek4 — APPROVED** [review]

File: src/modules/parseExtract.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+    }
+    // find the location of the end of the query
+    console.log(queryString);
+    // returns an array of the query and the location of the end of the query
+    return [queryString, count+1];
+  };
+
+  // Find queries inside of string, push them on to array
+  const findQuery = (string) => {
```
Comment (alexbednarek4) [inline src/modules/parseExtract.js]: What an absolute beauty.

**Review by alexbednarek4 — APPROVED** [review]
